lib/ipinfo.rb

```diff
@@ -142,6 +143,15 @@ def batch_requests(url_array, api_token)
     protected
 
     def request_details(ip_address = nil)
+        if isBogon(ip_address)
+            details[:ip] = ip_address
+            details[:bogon] = true
+            details[:ip_address] = IPAddr.new(ip_address)
+
+            @cache.set(cache_key(ip_address), details)
```

Comment:
I'd say there's no need to set in the cache for bogon IPs, since we'd never get from the cache and it'll just reduce capacity.